registration/registration-services/pom.xml

```diff
@@ -474,7 +481,7 @@
 		<dependency>
```

Comment:
We should remove stub-impl dependency from the pom, instead, this should be dynamically loaded when building the reg-client zip in the docker.

Reply:
Removed the dependency from here. Changes done to take the jar from artifactory-ref-impl